src/wp-content/themes/twentytwentyone/assets/sass/07-utilities/color-palette.scss

```diff
@@ -114,6 +114,10 @@
 	background-color: var(--global--color-black);
 }
 
+.has-black-border-color[class] {
+	border-color: var(--global--color-black, #000000);
```

Comment:
Why fallback colors for these?

Reply:
PostCSS complains without them:
```
variable --global--color-light-grey is undefined and used without a fallback [postcss-css-variables]
```

Reply:
If there needs to be fallback colors, kindly update them to be the correct hex colors. The colors from the palette.

Reply:
These proved to be unnecessary, I had a typo in one of the classnames: `grey` rather than `gray` and misinterpreted the warning message.